src/cli.yml

```diff
@@ -0,0 +1,42 @@
+name: dj-library-gain-calculator
+version: "0.1.0"
+about: Analyses all tracks in a Traktor DJ library to have constant loudness.
+subcommands:
+    - library:
```

Comment:
Call this collection, it's the term traktor uses

Reply:
Fixed.